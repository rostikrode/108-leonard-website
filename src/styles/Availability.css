.arrow-down-scroll {
  padding: 0.5em;
  display: none;
  position: absolute;
  bottom: 0;
  left: 50%;
  transform: translateX(-50%);
  transition: opacity 300ms ease-in-out;
  opacity: 1;
}
.arrow-down-scroll.hide {
  opacity: 0;
}

.availability-page {
  padding: 18px;
  position: relative;
}
.filter-button-wrapper {
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  align-items: center;
  position: fixed;
  top: 80px;
  left: 18px;
  width: 100%;
  background-color: #FFF;
  z-index: 800;
}
.filter-button-wrapper button {
  margin-top: 18px;
}
.filter-button-wrapper button:first-child {
  margin-right: 5px;
}

.list-wrapper {
  padding-top: 44px;
}

@media (min-width:1024px) {
  .arrow-down-scroll {
    display: block;
  }
  .availability-page {
    padding: 32px;
  }
  .filter-button-wrapper {
    top: 0;
    left: 0;
  }
  .filter-button-wrapper button {
    margin-top: 93px;
  }
  .filter-button-wrapper button:first-child {
    margin-left: 32px;
  }
  .list-wrapper {
    padding-top: 133px;
  }
}

@media (min-width:1366px) {
  .filter-button-wrapper {
    position: absolute;
  }
  .filter-button-wrapper button {
    margin-top: 32px;
  }
  .list-wrapper {
    padding-top: 50px;
  }
}