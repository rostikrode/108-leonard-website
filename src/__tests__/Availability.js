import 'babel-polyfill';
import React from 'react';
import ReactDOM from 'react-dom';
import { mount } from 'enzyme';
import fetch from 'isomorphic-fetch';
import Availability from '../components/pages/Availability';
import availJSON from '../components/data/availability.json'
import { MemoryRouter } from 'react-router-dom';


it('renders without crashing', () => {
  const tree = mount(
    <MemoryRouter><Availability {...availJSON}  /></MemoryRouter>
  );
  expect(tree.html()).toMatchSnapshot()
});

