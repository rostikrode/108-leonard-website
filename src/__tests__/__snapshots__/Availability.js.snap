// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `"<div class=\\"availability-page\\"><div class=\\"filter-button-wrapper\\"><button class=\\"button sans-light-bold   filter-button\\"><span>Filter</span></button><button class=\\"button sans-light-bold disabled  \\"><span>Share</span></button></div><div class=\\"list-wrapper\\"><div class=\\"filter\\"><div class=\\"floating-filter hide\\"><div class=\\"floating-filter-checkbox-wrapper\\"><div class=\\"floating-filter-group\\"><div class=\\"floating-filter-item\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"filter-checkbox-0\\" id=\\"checkbox-filter-checkbox-0\\" value=\\"1_bedrooms\\"><label for=\\"checkbox-filter-checkbox-0\\"></label></div><span class=\\"serif\\">1 Bedroom</span></div><div class=\\"floating-filter-item\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"filter-checkbox-1\\" id=\\"checkbox-filter-checkbox-1\\" value=\\"2_bedrooms\\"><label for=\\"checkbox-filter-checkbox-1\\"></label></div><span class=\\"serif\\">2 bedroom</span></div><div class=\\"floating-filter-item\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"filter-checkbox-2\\" id=\\"checkbox-filter-checkbox-2\\" value=\\"3_bedrooms\\"><label for=\\"checkbox-filter-checkbox-2\\"></label></div><span class=\\"serif\\">3 bedroom</span></div><div class=\\"floating-filter-item\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"filter-checkbox-3\\" id=\\"checkbox-filter-checkbox-3\\" value=\\"4_bedrooms\\"><label for=\\"checkbox-filter-checkbox-3\\"></label></div><span class=\\"serif\\">4 bedroom</span></div><div class=\\"floating-filter-item\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"filter-checkbox-4\\" id=\\"checkbox-filter-checkbox-4\\" value=\\"0_crowncollection\\"><label for=\\"checkbox-filter-checkbox-4\\"></label></div><span class=\\"serif\\">crown collection</span></div></div><div class=\\"floating-filter-group\\"><div class=\\"floating-filter-item\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"filter-checkbox-5\\" id=\\"checkbox-filter-checkbox-5\\" value=\\"1-2_price\\"><label for=\\"checkbox-filter-checkbox-5\\"></label></div><span class=\\"serif\\">$1-2M</span></div><div class=\\"floating-filter-item\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"filter-checkbox-6\\" id=\\"checkbox-filter-checkbox-6\\" value=\\"2-3_price\\"><label for=\\"checkbox-filter-checkbox-6\\"></label></div><span class=\\"serif\\">$2-3M</span></div><div class=\\"floating-filter-item\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"filter-checkbox-7\\" id=\\"checkbox-filter-checkbox-7\\" value=\\"3-4_price\\"><label for=\\"checkbox-filter-checkbox-7\\"></label></div><span class=\\"serif\\">$3-4M</span></div><div class=\\"floating-filter-item\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"filter-checkbox-8\\" id=\\"checkbox-filter-checkbox-8\\" value=\\"4-5_price\\"><label for=\\"checkbox-filter-checkbox-8\\"></label></div><span class=\\"serif\\">$4-5M</span></div><div class=\\"floating-filter-item\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"filter-checkbox-9\\" id=\\"checkbox-filter-checkbox-9\\" value=\\"5_price\\"><label for=\\"checkbox-filter-checkbox-9\\"></label></div><span class=\\"serif\\">$5M+</span></div></div></div><button class=\\"button sans-light-bold   \\"><span>View</span></button></div><div class=\\"header-filter list-row\\"><button data-filter=\\"residence\\" class=\\"filter-row list-cell\\"><span class=\\"sans filter-label\\">Residence</span><img src=\\"down_arrow_small.svg\\" alt=\\"downward arrow icon\\"></button><button data-filter=\\"bedrooms\\" class=\\"filter-row list-cell\\"><span class=\\"sans filter-label\\">Beds</span><img src=\\"down_arrow_small.svg\\" alt=\\"downward arrow icon\\"></button><button data-filter=\\"baths\\" class=\\"filter-row list-cell\\"><span class=\\"sans filter-label\\">Baths</span><img src=\\"down_arrow_small.svg\\" alt=\\"downward arrow icon\\"></button><button data-filter=\\"interior\\" class=\\"filter-row list-cell\\"><span class=\\"sans filter-label\\"><!-- react-text: 77 -->Interior <!-- /react-text --><br><!-- react-text: 79 -->sq ft/m<!-- /react-text --></span><img src=\\"down_arrow_small.svg\\" alt=\\"downward arrow icon\\"></button><button data-filter=\\"exterior\\" class=\\"filter-row list-cell\\"><span class=\\"sans filter-label\\"><!-- react-text: 83 -->Exterior <!-- /react-text --><br><!-- react-text: 85 -->sq ft/m<!-- /react-text --></span><img src=\\"down_arrow_small.svg\\" alt=\\"downward arrow icon\\"></button><button data-filter=\\"price\\" class=\\"filter-row list-cell\\"><span class=\\"sans filter-label\\">Price</span><img src=\\"down_arrow_small.svg\\" alt=\\"downward arrow icon\\"></button><button data-filter=\\"monthlycc\\" class=\\"filter-row list-cell hide-for-mobile flex\\"><span class=\\"sans filter-label\\"><!-- react-text: 92 -->est monthly <!-- /react-text --><br><!-- react-text: 94 -->c.c.<!-- /react-text --></span><img src=\\"down_arrow_small.svg\\" alt=\\"downward arrow icon\\"></button><button data-filter=\\"monthlytaxes\\" class=\\"filter-row list-cell hide-for-mobile flex\\"><span class=\\"sans filter-label\\">est monthly r.e. taxes</span><img src=\\"down_arrow_small.svg\\" alt=\\"downward arrow icon\\"></button><div class=\\"filter-row list-cell floorplan-space-keeper\\"></div></div></div><div class=\\"list\\"><div class=\\"list-row\\" id=\\"0\\"><div class=\\"list-row-wrapper\\"><div class=\\"list-cell check-cell\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"0\\" id=\\"checkbox-0\\" value=\\"A\\"><label for=\\"checkbox-0\\"></label></div><h4 class=\\"res serif-bold\\">A</h4></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BEDROOMS</p><p class=\\"info serif\\">2</p></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BATHROOMS</p><p class=\\"info serif\\">2.5</p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">INTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 117 -->1,783<!-- /react-text --><!-- react-text: 118 -->/<!-- /react-text --><!-- react-text: 119 -->165<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">EXTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 123 -->1,000<!-- /react-text --><!-- react-text: 124 -->/<!-- /react-text --><!-- react-text: 125 -->92<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">PRICE</p><p class=\\"info serif\\"><!-- react-text: 129 -->$<!-- /react-text --><!-- react-text: 130 -->1,000,000<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY C.C.</p><p class=\\"info serif\\"><!-- react-text: 134 -->$<!-- /react-text --><!-- react-text: 135 -->100<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY R.E. TAXES</p><p class=\\"info serif\\"><!-- react-text: 139 -->$<!-- /react-text --><!-- react-text: 140 -->100<!-- /react-text --></p></div><button data-id=\\"A\\" class=\\"button sans-light-bold  inverted list-cell floorplan-button desktop\\"><span>View Floorplan</span></button></div></div><div class=\\"list-row\\" id=\\"1\\"><div class=\\"list-row-wrapper\\"><div class=\\"list-cell check-cell\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"1\\" id=\\"checkbox-1\\" value=\\"B\\"><label for=\\"checkbox-1\\"></label></div><h4 class=\\"res serif-bold\\">B</h4></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BEDROOMS</p><p class=\\"info serif\\">2</p></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BATHROOMS</p><p class=\\"info serif\\">2.5</p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">INTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 159 -->1,443<!-- /react-text --><!-- react-text: 160 -->/<!-- /react-text --><!-- react-text: 161 -->134<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">EXTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 165 -->1,000<!-- /react-text --><!-- react-text: 166 -->/<!-- /react-text --><!-- react-text: 167 -->92<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">PRICE</p><p class=\\"info serif\\"><!-- react-text: 171 -->$<!-- /react-text --><!-- react-text: 172 -->1,000,000<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY C.C.</p><p class=\\"info serif\\"><!-- react-text: 176 -->$<!-- /react-text --><!-- react-text: 177 -->100<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY R.E. TAXES</p><p class=\\"info serif\\"><!-- react-text: 181 -->$<!-- /react-text --><!-- react-text: 182 -->100<!-- /react-text --></p></div><button data-id=\\"B\\" class=\\"button sans-light-bold  inverted list-cell floorplan-button desktop\\"><span>View Floorplan</span></button></div></div><div class=\\"list-row\\" id=\\"2\\"><div class=\\"list-row-wrapper\\"><div class=\\"list-cell check-cell\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"2\\" id=\\"checkbox-2\\" value=\\"C\\"><label for=\\"checkbox-2\\"></label></div><h4 class=\\"res serif-bold\\">C</h4></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BEDROOMS</p><p class=\\"info serif\\">3</p></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BATHROOMS</p><p class=\\"info serif\\">3.5</p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">INTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 201 -->1,922<!-- /react-text --><!-- react-text: 202 -->/<!-- /react-text --><!-- react-text: 203 -->178<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">EXTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 207 -->1,000<!-- /react-text --><!-- react-text: 208 -->/<!-- /react-text --><!-- react-text: 209 -->92<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">PRICE</p><p class=\\"info serif\\"><!-- react-text: 213 -->$<!-- /react-text --><!-- react-text: 214 -->1,000,000<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY C.C.</p><p class=\\"info serif\\"><!-- react-text: 218 -->$<!-- /react-text --><!-- react-text: 219 -->100<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY R.E. TAXES</p><p class=\\"info serif\\"><!-- react-text: 223 -->$<!-- /react-text --><!-- react-text: 224 -->100<!-- /react-text --></p></div><button data-id=\\"C\\" class=\\"button sans-light-bold  inverted list-cell floorplan-button desktop\\"><span>View Floorplan</span></button></div></div><div class=\\"list-row\\" id=\\"3\\"><div class=\\"list-row-wrapper\\"><div class=\\"list-cell check-cell\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"3\\" id=\\"checkbox-3\\" value=\\"E\\"><label for=\\"checkbox-3\\"></label></div><h4 class=\\"res serif-bold\\">E</h4></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BEDROOMS</p><p class=\\"info serif\\">2</p></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BATHROOMS</p><p class=\\"info serif\\">2.5</p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">INTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 243 -->1,306<!-- /react-text --><!-- react-text: 244 -->/<!-- /react-text --><!-- react-text: 245 -->121<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">EXTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 249 -->1,000<!-- /react-text --><!-- react-text: 250 -->/<!-- /react-text --><!-- react-text: 251 -->92<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">PRICE</p><p class=\\"info serif\\"><!-- react-text: 255 -->$<!-- /react-text --><!-- react-text: 256 -->1,000,000<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY C.C.</p><p class=\\"info serif\\"><!-- react-text: 260 -->$<!-- /react-text --><!-- react-text: 261 -->100<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY R.E. TAXES</p><p class=\\"info serif\\"><!-- react-text: 265 -->$<!-- /react-text --><!-- react-text: 266 -->100<!-- /react-text --></p></div><button data-id=\\"E\\" class=\\"button sans-light-bold  inverted list-cell floorplan-button desktop\\"><span>View Floorplan</span></button></div></div><div class=\\"list-row\\" id=\\"4\\"><div class=\\"list-row-wrapper\\"><div class=\\"list-cell check-cell\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"4\\" id=\\"checkbox-4\\" value=\\"F\\"><label for=\\"checkbox-4\\"></label></div><h4 class=\\"res serif-bold\\">F</h4></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BEDROOMS</p><p class=\\"info serif\\">2</p></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BATHROOMS</p><p class=\\"info serif\\">2.5</p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">INTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 285 -->1,536<!-- /react-text --><!-- react-text: 286 -->/<!-- /react-text --><!-- react-text: 287 -->142<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">EXTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 291 -->1,000<!-- /react-text --><!-- react-text: 292 -->/<!-- /react-text --><!-- react-text: 293 -->92<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">PRICE</p><p class=\\"info serif\\"><!-- react-text: 297 -->$<!-- /react-text --><!-- react-text: 298 -->1,000,000<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY C.C.</p><p class=\\"info serif\\"><!-- react-text: 302 -->$<!-- /react-text --><!-- react-text: 303 -->100<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY R.E. TAXES</p><p class=\\"info serif\\"><!-- react-text: 307 -->$<!-- /react-text --><!-- react-text: 308 -->100<!-- /react-text --></p></div><button data-id=\\"F\\" class=\\"button sans-light-bold  inverted list-cell floorplan-button desktop\\"><span>View Floorplan</span></button></div></div><div class=\\"list-row\\" id=\\"5\\"><div class=\\"list-row-wrapper\\"><div class=\\"list-cell check-cell\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"5\\" id=\\"checkbox-5\\" value=\\"G\\"><label for=\\"checkbox-5\\"></label></div><h4 class=\\"res serif-bold\\">G</h4></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BEDROOMS</p><p class=\\"info serif\\">3</p></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BATHROOMS</p><p class=\\"info serif\\">3.5</p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">INTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 327 -->2,015<!-- /react-text --><!-- react-text: 328 -->/<!-- /react-text --><!-- react-text: 329 -->187<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">EXTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 333 -->1,000<!-- /react-text --><!-- react-text: 334 -->/<!-- /react-text --><!-- react-text: 335 -->92<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">PRICE</p><p class=\\"info serif\\"><!-- react-text: 339 -->$<!-- /react-text --><!-- react-text: 340 -->1,000,000<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY C.C.</p><p class=\\"info serif\\"><!-- react-text: 344 -->$<!-- /react-text --><!-- react-text: 345 -->100<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY R.E. TAXES</p><p class=\\"info serif\\"><!-- react-text: 349 -->$<!-- /react-text --><!-- react-text: 350 -->100<!-- /react-text --></p></div><button data-id=\\"G\\" class=\\"button sans-light-bold  inverted list-cell floorplan-button desktop\\"><span>View Floorplan</span></button></div></div><div class=\\"list-row\\" id=\\"6\\"><div class=\\"list-row-wrapper\\"><div class=\\"list-cell check-cell\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"6\\" id=\\"checkbox-6\\" value=\\"H\\"><label for=\\"checkbox-6\\"></label></div><h4 class=\\"res serif-bold\\">H</h4></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BEDROOMS</p><p class=\\"info serif\\">2</p></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BATHROOMS</p><p class=\\"info serif\\">2.5</p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">INTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 369 -->1,584<!-- /react-text --><!-- react-text: 370 -->/<!-- /react-text --><!-- react-text: 371 -->147<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">EXTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 375 -->1,000<!-- /react-text --><!-- react-text: 376 -->/<!-- /react-text --><!-- react-text: 377 -->92<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">PRICE</p><p class=\\"info serif\\"><!-- react-text: 381 -->$<!-- /react-text --><!-- react-text: 382 -->1,000,000<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY C.C.</p><p class=\\"info serif\\"><!-- react-text: 386 -->$<!-- /react-text --><!-- react-text: 387 -->100<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY R.E. TAXES</p><p class=\\"info serif\\"><!-- react-text: 391 -->$<!-- /react-text --><!-- react-text: 392 -->100<!-- /react-text --></p></div><button data-id=\\"H\\" class=\\"button sans-light-bold  inverted list-cell floorplan-button desktop\\"><span>View Floorplan</span></button></div></div><div class=\\"list-row\\" id=\\"7\\"><div class=\\"list-row-wrapper\\"><div class=\\"list-cell check-cell\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"7\\" id=\\"checkbox-7\\" value=\\"L\\"><label for=\\"checkbox-7\\"></label></div><h4 class=\\"res serif-bold\\">L</h4></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BEDROOMS</p><p class=\\"info serif\\">2</p></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BATHROOMS</p><p class=\\"info serif\\">2.5</p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">INTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 411 -->1,472<!-- /react-text --><!-- react-text: 412 -->/<!-- /react-text --><!-- react-text: 413 -->136<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">EXTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 417 -->1,000<!-- /react-text --><!-- react-text: 418 -->/<!-- /react-text --><!-- react-text: 419 -->92<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">PRICE</p><p class=\\"info serif\\"><!-- react-text: 423 -->$<!-- /react-text --><!-- react-text: 424 -->1,000,000<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY C.C.</p><p class=\\"info serif\\"><!-- react-text: 428 -->$<!-- /react-text --><!-- react-text: 429 -->100<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY R.E. TAXES</p><p class=\\"info serif\\"><!-- react-text: 433 -->$<!-- /react-text --><!-- react-text: 434 -->100<!-- /react-text --></p></div><button data-id=\\"L\\" class=\\"button sans-light-bold  inverted list-cell floorplan-button desktop\\"><span>View Floorplan</span></button></div></div><div class=\\"list-row\\" id=\\"8\\"><div class=\\"list-row-wrapper\\"><div class=\\"list-cell check-cell\\"><div class=\\"checkbox\\"><input type=\\"checkbox\\" name=\\"8\\" id=\\"checkbox-8\\" value=\\"M\\"><label for=\\"checkbox-8\\"></label></div><h4 class=\\"res serif-bold\\">M</h4></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BEDROOMS</p><p class=\\"info serif\\">2</p></div><div class=\\"mobile-padding list-cell label-with-info\\"><p class=\\"label sans\\">BATHROOMS</p><p class=\\"info serif\\">2.5</p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">INTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 453 -->1,809<!-- /react-text --><!-- react-text: 454 -->/<!-- /react-text --><!-- react-text: 455 -->168<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">EXTERIOR SQ FT/M</p><p class=\\"info serif\\"><!-- react-text: 459 -->1,000<!-- /react-text --><!-- react-text: 460 -->/<!-- /react-text --><!-- react-text: 461 -->92<!-- /react-text --></p></div><div class=\\"list-cell label-with-info\\"><p class=\\"label sans\\">PRICE</p><p class=\\"info serif\\"><!-- react-text: 465 -->$<!-- /react-text --><!-- react-text: 466 -->1,000,000<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY C.C.</p><p class=\\"info serif\\"><!-- react-text: 470 -->$<!-- /react-text --><!-- react-text: 471 -->100<!-- /react-text --></p></div><div class=\\"list-cell label-with-info hide-for-mobile\\"><p class=\\"label sans\\">EST MONTHLY R.E. TAXES</p><p class=\\"info serif\\"><!-- react-text: 475 -->$<!-- /react-text --><!-- react-text: 476 -->100<!-- /react-text --></p></div><button data-id=\\"M\\" class=\\"button sans-light-bold  inverted list-cell floorplan-button desktop\\"><span>View Floorplan</span></button></div></div></div></div><img src=\\"down_arrow_large.svg\\" class=\\"arrow-down-scroll\\" alt=\\"downward arrow icon\\"><div class=\\"floorplan-overlay show\\"><button class=\\"close-btn\\"><img src=\\"close_blue.svg\\" alt=\\"close btn\\" class=\\"close-btn-img\\" width=\\"25\\" height=\\"25\\"></button><a class=\\"button download-btn desktop sans-light-bold\\" target=\\"_blank\\" href=\\"/images/5_availability/pdfs/residence_undefined.pdf\\">DOWNLOAD</a><div class=\\"floorplan-content\\"><div class=\\"floorplan-sidebar\\"><div class=\\"floorplan-info\\"><div class=\\"floorplan-title sans-medium\\"><!-- react-text: 488 -->RESIDENCE <!-- /react-text --></div><div class=\\"floorplan-bedrooms serif\\">undefined BEDROOMS</div><div class=\\"floorplan-bathrooms serif\\">undefined  BATHROOMS</div><div class=\\"floorplan-interior serif\\"><!-- react-text: 492 -->undefined<!-- /react-text --><!-- react-text: 493 --> SQ FT | <!-- /react-text --><!-- react-text: 494 -->NaN<!-- /react-text --><!-- react-text: 495 --> SQ M<!-- /react-text --></div><div class=\\"floorplan-exposure serif\\"><!-- react-text: 497 --> <!-- /react-text --><br><!-- react-text: 499 --> EXPOSURES<!-- /react-text --></div></div><div class=\\"floorplan-keys\\"><img src=\\"/images/5_availability/keys/residence_undefined_key.svg\\" alt=\\"Floor Layout for Residence undefined\\"></div></div><div class=\\"floorplan-wrapper\\" id=\\"residence-undefined\\"><p class=\\"dbxd-no-print\\">Printing these floorplans is not allowed.</p><div class=\\"dbxd-floorplan-layout\\"><div class=\\"dbxd-floorplan-wrapper\\"><span class=\\"react-draggable\\"><div class=\\"dbxd-floorplan desktop-fp\\" style=\\"background-image: url(/images/5_availability/floorplans/residence_undefined.svg);\\"></div><div class=\\"loading-wrapper\\"><i class=\\"loading\\"></i></div></span><div class=\\"dbxd-tutorial-wrapper show\\"><div class=\\"dbxd-tutorial-list\\"><button title=\\"close info window button\\" class=\\"dbxd-close\\"><div class=\\"loading-wrapper\\"><i class=\\"loading\\"></i></div></button><div class=\\"sans dbxd-feat zoom\\">Click floorplan to zoom in and out.</div><div class=\\"sans dbxd-feat pan_clickdrag\\"><!-- react-text: 517 -->While zoomed in, click and drag mouse to pan the floorplan.<!-- /react-text --><br><br><br><!-- react-text: 521 -->On mobile and tablet devices, pinch your screen to zoom in.<!-- /react-text --></div></div></div></div><button title=\\"Show Intructions\\" class=\\"dbxd-instructions-icon\\"><div class=\\"loading-wrapper\\"><i class=\\"loading\\"></i></div></button><p class=\\"dbxd-copyright\\"><a href=\\"//www.dbxd.com\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\" class=\\"sans-medium\\">© DBXD</a></p></div></div></div></div></div>"`;
